### DISCUSSION TITLE
Review feature for prompt deployment

### DISCUSSION THREAD

**Discussion by tamdoEH** [langfuse/langfuse#1710] [Ideas]
### Describe the feature or potential improvement

It would be beneficial for teams to have some pull request feature, where each prompt deployed to production needs at least 1 review.

**Comment by marcklingen** [comment]
Alternative to this could be to have roles which can only add prompts but cannot attach the production label / activate them.

**Comment by ZugunruheKami** [comment]
Thanks for the idea and alternative simpler implementation by @marcklingen. would be great to have this auditability alongside the current testability suites working in large teams

**Comment by kftam1994** [comment]
It would be auditable if the name of the person who attach the production label can be shown in the UI.
